AppController/lib/taskqueue.rb

```diff
@@ -76,11 +80,21 @@ def self.start_rabbitmq
        raise AppScaleException.new(msg)
     end
 
+    pidfile = File.join('/', 'var', 'lib', 'rabbitmq', 'mnesia',
+                        "rabbit@#{Socket.gethostname}.pid")
+    begin
+      rabbitmq_version = self.get_rabbitmq_version
+      if rabbitmq_version[0] <= 3 && rabbitmq_version[1] < 4
```

Comment:
It seems that statement won't work as expected for 2.8.* (for example).
`if rabbitmq_version[0] < 3 || (rabbitmq_version[0] == 3 && rabbitmq_version[1] < 4)`